sdv/metadata/multi_table.py

```diff
@@ -776,9 +776,11 @@ def _validate_all_tables(self, data):
                     self.tables[table_name].validate_data(table_data, table_sdtype_warnings)
 
             except InvalidDataError as error:
-                error_msg = f"Table: '{table_name}'"
-                for _error in error.errors:
-                    error_msg += f'\nError: {_error}'
+                error_msg = '' if len(self.tables) == 1 else f"Table: '{table_name}'\n"
```

Comment:
From what I remember in the eng meeting, I think we said to change this to something that is fine in both cases. For example
```
Errors in {table_name}:
```
If the above happens to be the default single table case, it will read
```
Errors in table:
```
Which is fine.